client/client.go

```diff
@@ -586,53 +439,9 @@ func (c *CfClient) Environment() string {
 }
 
 // InterceptAddCluster adds cluster ID to calls
-func (c *CfClient) InterceptAddCluster(ctx context.Context, req *http.Request) error {
+func (c *CfClient) InterceptAddCluster(_ context.Context, req *http.Request) error {
```

Comment:
Having an '_' feels wrong.
I'm guessing this is because its a public function and we don't want to alter the API?

Reply:
thats why I put do not merge label, some things are unclear for this ticket and I dont like warnings